nuspec/MvvmCross.Console.Platform.nuspec

```diff
@@ -15,17 +15,7 @@ This package contains the Console 'MvvmCross Library' assemblies for MvvmCross
 		</dependencies>
 	</metadata>
 	<files>		
-		<!-- PCL -->
-		<file src="..\bin\Release\Mvx\Portable\MvvmCross.Core.*" target="lib\portable-net45+win+wpa81+wp80" />
-
-		<!-- dotnet -->
-		<file src="..\bin\Release\Mvx\Portable\MvvmCross.Core.*" target="lib\dotnet" />
-		
 		<!-- console -->
 		<file src="..\bin\Release\Mvx\Console\MvvmCross.Console.*" target="lib\net45" />
```

Comment:
Sorry that comment applied to the inclusion of `Console.*`.  I just didn't expect it.